libs/common/src/helpers/is-number-generator-mocked.ts

```diff
@@ -0,0 +1,15 @@
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+/**
+ * checks if number generator key is mocked
+ * if numberGeneratorKey is mocked, it will have the same value as MOCK_E2E_NUMBER_GENERATOR
+ * @returns true if numberGeneratorKey is the same as MOCK_E2E_NUMBER_GENERATOR
+ */
+export const isNumberGeneratorMocked = (): boolean => {
+  const numberGeneratorKey: string = process.env.NUMBER_GENERATOR ?? '';
+  const mockNumberGeneratorKey: string = process.env.MOCK_E2E_NUMBER_GENERATOR ?? '';
+
+  return numberGeneratorKey === mockNumberGeneratorKey;
```

Comment:
```suggestion
  return mockNumberGeneratorKey === 'MOCKED';
```

Since numgen does not have a key, same implementation as GovNotify won't work, we will need a hard coded mocked value please.

Reply:
have added validation that if either is not defined or empty, then return false, so should not need this then?